buildpack.toml

```diff
@@ -41,6 +41,12 @@ api = "0.7"
     description = "the application context path"
     name = "BP_TOMCAT_CONTEXT_PATH"
 
+  [[metadata.configurations]]
+    build = true
+    description = "the application server to use"
```

Comment:
You may want to reorder these properties or you'll end up with a lot of empty PRs created by the update jobs.